cpp/csp/cppnodes/statsimpl.h

```diff
@@ -1,11 +1,16 @@
 #include <csp/engine/CppNode.h>
 #include <csp/engine/WindowBuffer.h>
-#include <ext/pb_ds/assoc_container.hpp>
-#include <ext/pb_ds/tree_policy.hpp>
+
 #include <functional>
 #include <numeric>
+#include <set>
 #include <type_traits>
 
+#ifdef __GNUC__
```

Comment:
Unfortunately `clang` also defines these so better to invert for now
```suggestion
#ifndef __clang__
```

Reply:
Fixed